src/test/java/org/apache/commons/csv/CSVParserTest.java

```diff
@@ -572,7 +572,7 @@ public void testGetOneLine() throws IOException {
     /**
      * Tests reusing a parser to process new string records one at a time as they are being discovered. See [CSV-110].
      *
-     * @throws IOException
+     * @throws IOException If something goes wrong
```

Comment:
Might as well use the "standard" comment here

Reply:
Sorry Garry, but What is the standard comment?

Reply:
@arturobernalg 
The JRE uses "if an I/O error occurs." but I prefer "Thrown when an I/O error occurs."

Reply:
ok, got it. Change it.